app/javascript/shared/helpers/vuex/mutationHelpers.js

```diff
@@ -45,6 +45,16 @@ export const updatePresence = (state, data) => {
   });
 };
 
+export const updateSingleAgentPresence = (
```

Comment:
Since this is resource specific we could move it helpers under agents module?

Reply:
@nithindavid `updatePresence` is already part of MutationHelper as it is used in multiple places. It makes sense to have it here rather than moving it to the agents module. A generic name, if needed, can be used.

Reply:
Renamed it to updateSingleRecordPresence